dadi/lib/model/index.js

```diff
@@ -112,11 +112,22 @@ Model.prototype.create = function (obj, internals, done) {
 
     // apply any existing `beforeCreate` hooks
     if (typeof this.settings.hooks.beforeCreate === 'object') {
+      if (obj instanceof Array) {
```

Comment:
@eduardoboucas you can see I've had to duplicate the hook apply method here, when we have been passed an array of documents. 

The same has been done for the `afterCreate` hooks.

If you can make this look actually tidy, we'll be all set to merge :+1: 

Thanks!